### ISSUE TITLE
Minor: Version is not updated in the Jar file - still at 0.8.2

### ISSUE THREAD

**Issue by SnoCold** [sylvainhalle/textidote#216]
Hi,
Not a big issue but it confused me a little

the version in the jar file still states 0.8.2 instead of 0.8.3

```console
PS C:\Users\nishshuk\temop> java -jar ..\Downloads\textidote.jar --help
TeXtidote v0.8.2 - A linter for LaTeX documents and others
(C) 2018-2021 Sylvain HallΘ - All rights reserved

Usage: java -jar textidote.jar [options] file1 [file2 ...]

--name n        Use n as app name when printing usage
--check lang    Checks grammar in language lang
--firstlang lang        Checks for false friends with the author's first language lang and the language specified in --check
--remove-macros macs    Remove LaTeX macros macs
--replace file  Apply replacement patterns from file
--version       Show version number
--clean         Remove markup from input file
--dict file     Load dictionary from file
--help          Show command line usage
--output method Output as plain (default), json, html, or singleline
--encoding x    Read files using encoding x
--read-all      Don't ignore lines before \begin{document}
--ignore rules  Ignore rules
--languagemodel dir     Use n-grams data from dir
--ci    Ignores the return code for CI usage
--type x        Input is of type x (tex or md)
--map file      Output correspondence map to file
--no-color      Disables colors in ANSI printing
--quiet         Don't print any message
--no-config     Ignore config file if any
--remove envs   Remove LaTeX environments envs

```

**Comment by sylvainhalle** [comment]
Oops! I'll try to recompile and overwrite the current release.

**Comment by SnoCold** [comment]
Thank you so much, sorry for the trouble 👍

**Comment by sylvainhalle** [comment]
The last modifications in the code should fix the issue for release 0.9
